dice.js

```diff
@@ -0,0 +1,26 @@
+const random = () => {
+  return Math.round(Math.random() * 5 + 1);
+};
+
+const rollDice = (x = 2) => {
+  let allDices = [];
+
+  for (let i = 0; i < x; i++) {
+    let randomNumber = random();
+    allDices.push(randomNumber);
+  }
+
+  let allDicesString = allDices
+    .slice(0, x - 1)
+    .toString()
+    .replace(/,/g, ", ");
```

Comment:
Great work taking the commas and spacing into consideration.
you can make the code even shorter by replacing `toString().replace(/,/g, ", ");`  with  `join(", ")` as it will give the same result.